src/scenes/home/header/sideNav/sideNav.css

```diff
@@ -66,5 +66,14 @@
   text-align: left;
   font-size: 2.5rem;
   margin: 0.5rem 0;
-  padding: 0; 
+  padding: 0;
 }
+
+@media screen and (orientation: landscape) {
```

Comment:
Oh, you *fancy* with these media queries...

Question: Orientation affects iPads too - did you check to see how this affects tablets on Browserstack?